docu/release_notes/2016-12-06-Technev153.html.md

```diff
@@ -0,0 +1,15 @@
+---
+date: December 20, 2016
+service: 'hybris techne'
+official_version: v1.5.3
+headline: 'Avatars and Images Pattern'
+---
+
+### New Pattern
+* New pattern that that displays avatars and images on different context. Markup and iteraction on how to Upload/Change images.
```

Comment:
There are a few items to address here: 
- Don't use bullet points for one item. 
- Write in complete sentences, as in, "A new pattern displays avatars and images in different contexts." 
- When we say "different context" what do we mean? That, depending on a user's role when signed in, they can display different images? 
- Also, the second sentence does not make sense. What does it mean? There is markup somewhere and something interactive that tells how to Upload and change images?